salt/modules/drbd.py

```diff
@@ -20,9 +21,117 @@ def _analyse_overview_field(content):
     return content, ""
 
 
+def _count_spaces(line):
```

Comment:
This doesn't seem like it does what I would expect it to. Ex:

```
def counter():
    line = "a b c"
    spaces = 0
    for i in line:
        print(i)
        if i.isspace():
            spaces += 1
        else:
            return spaces

spaces = counter()
print("Found {0} spaces".format(spaces))
```

```
gray% python ./foo.py
a
Found 0 spaces
```

Reply:
It is my bad of naming the function, the idea is suppose to count the number of spaces before the first alphabeta character.
The output of "drbdadm status" doesn't have a standard format, since it may have multiple resources, each resources may have multiple peer nodes or volumes. So it use two spaces in front of a line as an indentation to figure out the section.

I will rename the function like _count_spaces_startswith()